bluemira/builders/EUDEMO/plasma.py

```diff
@@ -116,7 +117,7 @@ class PlasmaBuilder(Builder):
     _plot_flag: bool
     _segment_angle: float
     _eqdsk_path: Optional[str] = None
-    _default_run_mode: str = "run"
+    _default_runmode: str = "run"
```

Comment:
None of the kwargs in this file are needed at the moment I dont think, tell me if I'm wrong